test/integration/rule-fault-injection.yaml.tmpl

```diff
@@ -1,19 +0,0 @@
-destination: {{.destination}}.{{.namespace}}.svc.cluster.local
-precedence: 3
-match:
-  source: {{.source}}.{{.namespace}}.svc.cluster.local
-  source_tags:
-    version: v1
-  httpHeaders:
-    version:
-      exact: v2
-route:
-  - tags:
-       version: v2
-http_fault:
```

Comment:
I see http_fault has been renamed to httpFault. Is this correct?

It would be good to do git mv first and then make the change.